webpack.config.js

```diff
@@ -60,17 +60,19 @@ if (NPM_RUN === 'build' || NPM_RUN === 'start') {
     filename: 'dash-shaka-playback.js',
     plugins: [],
     externals: externals(),
-    mode: 'development'
+    mode: 'development',
+    port: '8181',
   }))
 
   // Unminified bundle without shaka-player
-  var customExt = externals()
+  let customExt = externals()
```

Comment:
You can use `const` if you are not going to reassign `customExt`
```suggestion
  const customExt = externals()
```